pytket/binders/circuit_library.cpp

```diff
@@ -22,6 +22,9 @@ namespace py = pybind11;
 namespace tket {
 
 PYBIND11_MODULE(circuit_library, library_m) {
+  library_m.def(
+      "BRIDGE_using_CX_0", &CircPool::BRIDGE_using_CX_0,
+      "Equivalent to BRIDGE, using four CX, first CX has control on qubit 0");
   library_m.def(
       "_BRIDGE_using_CX_0", &CircPool::BRIDGE_using_CX_0,
```

Comment:
I checked that this isn't used in any of the extensions. So I think we can remove the underscored version.